.rat-excludes

```diff
@@ -19,10 +19,9 @@
 **/*.json
 **/*.prefs
 **/*.log
-**/*.md
```

Comment:
We don't need to add the license header on the markdown file.

Reply:
> We don't need to add the license header on the markdown file.

Got it, in fact the reason we add the headers is from the feedback of the original VOTE mail (https://lists.apache.org/thread/k76sopyhwrfp32mcgczfqf0086g86nro). We have deleted the headers, please have a check, thanks!

Reply:
It looks good to me now.

Reply:
> 

Thanks! We will restart the release process soon.